lms/static/scripts/frontend_apps/components/BookPicker.js

```diff
@@ -30,6 +30,10 @@ import ErrorDisplay from './ErrorDisplay';
  * which chapter to use from that book. Once both are chosen the `onSelectBook`
  * callback is called.
  *
+ * Note: The component logic is consistent about using the terminology of
+ * "chapter" for content segments in a VitalSource eBook, but presents this
+ * to the user as "location" and other slightly less precise language.
```

Comment:
We might want to schedule a technical task to rename references to "chapter" in the code to something that better matches the terminology used in the UI, to avoid confusion for future devs.